lib/papertrail/http_client.rb

```diff
@@ -46,9 +46,8 @@ def token_auth(token)
     end
 
     def get(path, params = {})
-      if params.size > 0
-        path = "#{path}?cli_version=#{Papertrail::VERSION}&#{build_nested_query(params)}"
-      end
+      params.merge!(cli_version)
+      path = "#{path}?#{build_nested_query(params)}"
```

Comment:
@armchairlinguist went rogue in https://github.com/papertrail/papertrail-cli/pull/88/commits/199a95ba54a3c9682791b729bcc707923a820806